_includes/head.html

```diff
@@ -16,4 +16,16 @@
   <link rel="manifest" href="{{ site.url }}/manifest.json">
   <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="{{ site.url }}/feed.xml">
   <title>{% if page.title %}{{ page.title }} - {% endif %}{{ site.title }}</title>
-</head>
+  <meta property="og:title" content="{% if page.title %}{{ page.title }} - {% endif %}{{ site.title }}" />
+  <meta property="og:site_name" content="{% if page.title %}{{ page.title }} - {% endif %}{{ site.title }}"/>
```

Comment:
Perhaps it makes sense to wrap the output of `<meta property="og:title">` in the `{% if page.title %}` check.